.github/workflows/build.yaml

```diff
@@ -57,46 +55,17 @@ jobs:
               env:
                   CONCURRENCY_CONTEXT: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
               run: echo "$CONCURRENCY_CONTEXT"
-            - uses: Wandalen/wretry.action@v1.3.0
-              name: Checkout
-              with:
-                  action: actions/checkout@v3.5.2
-                  with: |
-                      token: ${{ github.token }}
-                  attempt_limit: 3
-                  attempt_delay: 2000
-            - name: Checkout submodules
-              run: scripts/checkout_submodules.py --allow-changing-global-git-config --shallow --platform linux
+            - name: Checkout
+              uses: actions/checkout@v3
```

Comment:
Again, why is this removing the retry?  And other places too.

Reply:
It was an infra issue (firewall config at github), which has since been resolved, and no reports of it for the last year.

Reply:
Note: I also did shotgun audit of this command, and it never kicked in once.